opensourceleg/actuators/base.py

```diff
@@ -0,0 +1,423 @@
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    List,
+    NamedTuple,
+    Optional,
+    Protocol,
+    Set,
+    TypeVar,
+    Union,
+    cast,
+    runtime_checkable,
+)
+
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from enum import Enum
+from functools import partial
+
+import numpy as np
+
+from opensourceleg.logging.logger import LOGGER
+
+
+class MOTOR_CONSTANTS(NamedTuple):
+    # TODO: Add thermal constants
+    MOTOR_COUNT_PER_REV: float
+    NM_PER_AMP: float
+    NM_PER_RAD_TO_K: float
```

Comment:
What does `NM_PER_RAD_TO_K` mean? Shouldn't that just be 1?

Reply:
Just realizing how dumb this name sounds haha. This constant corresponds to the conversion from Nm/rad to PID units.

Dephy's PID variables are named "k" and "b" so I just named it this way without thinking about it. Please feel free to suggest a better name for these two constants.

Reply:
A possible suggestion: 

NM_PER_RAD_TO_K -> K_SCALING

NM_S_PER_RAD_TO_B -> B_SCALING